src/screens/Learning.js

```diff
@@ -182,7 +225,11 @@ export default ({
               <NextButton
                 isDisabled={false}
                 textColor="#FFFFFF"
-                text={'Reshuffle'}
+                text={
+                  nativeLanguage === LANGUAGE_NAMES.EN
+                    ? 'Reshuffle'
+                    : 'Atsombadika'
+                }
```

Comment:
use LANG_DATA for all the strings in the app

Reply:
All comments resolved.